components/Banner/Banner.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component, PropTypes } from 'react'
+import s from './Banner.css'
+
+const goodTitle = 'Success!'
+const badTitle = 'Something went wrong!'
+
+const formatBanner = (props) => {
```

Comment:
I wonder if we should also optionally pass in the titles so they can be more specific to what succeeded or failed. Not a blocker though

Reply:
That's a good idea.  It's generally more flexible that way, too.  I'll add it to the scrum board though, since I'd like to ship this ASAP.